go.mod

```diff
@@ -14,24 +14,24 @@ replace k8s.io/client-go => k8s.io/client-go v0.22.1
 replace github.com/russellhaering/goxmldsig@v1.1.0 => github.com/russellhaering/goxmldsig v1.1.1
 
 require (
-	cloud.google.com/go/storage v1.14.0
+	cloud.google.com/go/storage v1.18.2
 	cuelang.org/go v0.4.0
-	github.com/Azure/azure-sdk-for-go v57.1.0+incompatible
+	github.com/Azure/azure-sdk-for-go v59.3.0+incompatible
```

Comment:
@sunker @marefr  -- Do the updates here seem reasonable?

We are importing `gocloud.dev/blob` -- that has up-to-date deps with the major cloud providers, so I think it is OK, but want to make sure!

Reply:
Should be fine, thank you!